### PR TITLE
1907-04-22-Richardson 

### PR DESCRIPTION
**Pull request by claudia5**
Monday issue page 1 for deliverable

### CODE REVIEW

File: 1907-04-22.xml [hunk 1]
```diff
@@ -0,0 +1,1542 @@
...
+                            <cell><date when="1907-03-05">5</date>, <date when="1907-03-12"
+                                >12</date>, <date when="1907-03-19">19</date>, <date
+                                    when="1907-03-26">26 March</date>.</cell>
+                            <cell>….….….</cell>
+                        </row>
+                        <row>
+                            <cell><date when="1907-04-06">6</date>, <date when="1907-04-13">13
+                                April</date>.</cell>
+                            <cell><date when="1907-04-02">2</date>, <date when="1907-04-09"
+                                >9</date>, <date when="1907-04-16">16 April</
```
Comment (whanley) [inline 1907-04-22.xml]: One of these `</div>` tags should go to the end of the page.

**Comment by whanley** [comment]
Good fix on the `</div>` tag. Dates still remain to be fixed.

**Comment by whanley** [comment]
Good work. I'll call this complete, but there's still one little detail to fix. The December dates you give as 1907-12, but they're probably 1906-12 (i.e. from the previous year). What do you think?

**Comment by claudia5** [comment]
I agree, I will make those changes before turning in the next deliverable.

**Pull request merged**